4) readFromUrl(com.google.common.jimfs.UrlTest)
java.nio.file.FileSystemNotFoundException: Provider "jimfs" not installed
        at java.base/java.nio.file.Path.of(Path.java:212)
        at java.base/java.nio.file.Paths.get(Paths.java:98)
        at com.google.common.jimfs.PathURLConnection.connect(PathURLConnection.java:76)
        at com.google.common.jimfs.PathURLConnection.getInputStream(PathURLConnection.java:122)
        at java.base/java.net.URL.openStream(URL.java:1161)
        at com.google.common.io.Resources$UrlByteSource.openStream(Resources.java:70)
        at com.google.common.io.ByteSource.read(ByteSource.java:296)
        at com.google.common.io.ByteSource$AsCharSource.read(ByteSource.java:486)
        at com.google.common.jimfs.UrlTest.readFromUrl(UrlTest.java:60)
5) readDirectoryContents(com.google.common.jimfs.UrlTest)
java.nio.file.FileSystemNotFoundException: Provider "jimfs" not installed
        at java.base/java.nio.file.Path.of(Path.java:212)
        at java.base/java.nio.file.Paths.get(Paths.java:98)
        at com.google.common.jimfs.PathURLConnection.connect(PathURLConnection.java:76)
        at com.google.common.jimfs.PathURLConnection.getInputStream(PathURLConnection.java:122)
        at java.base/java.net.URL.openStream(URL.java:1161)
        at com.google.common.io.Resources$UrlByteSource.openStream(Resources.java:70)
        at com.google.common.io.ByteSource.read(ByteSource.java:296)
        at com.google.common.io.ByteSource$AsCharSource.read(ByteSource.java:486)
        at com.google.common.jimfs.UrlTest.readDirectoryContents(UrlTest.java:72)
6) contentType(com.google.common.jimfs.UrlTest)
java.nio.file.FileSystemNotFoundException: Provider "jimfs" not installed
        at java.base/java.nio.file.Path.of(Path.java:212)
        at java.base/java.nio.file.Paths.get(Paths.java:98)
        at com.google.common.jimfs.PathURLConnection.connect(PathURLConnection.java:76)
        at com.google.common.jimfs.PathURLConnection.getHeaderField(PathURLConnection.java:140)
        at java.base/java.net.URLConnection.getContentType(URLConnection.java:520)
        at com.google.common.jimfs.UrlTest.contentType(UrlTest.java:124)
7) headers(com.google.common.jimfs.UrlTest)
java.nio.file.FileSystemNotFoundException: Provider "jimfs" not installed
        at java.base/java.nio.file.Path.of(Path.java:212)
        at java.base/java.nio.file.Paths.get(Paths.java:98)
        at com.google.common.jimfs.PathURLConnection.connect(PathURLConnection.java:76)
        at com.google.common.jimfs.PathURLConnection.getHeaderFields(PathURLConnection.java:130)
        at com.google.common.jimfs.UrlTest.headers(UrlTest.java:85)
--- a/src/test/java/com/google/common/jimfs/UrlTest.java
+++ b/src/test/java/com/google/common/jimfs/UrlTest.java
@@ -32,6 +32,7 @@ import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.attribute.FileTime;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
@@ -52,7 +53,7 @@ public class UrlTest {
     assertThat(url).isNotNull();
   }
 
-  @Test
+  @Test @Ignore
   public void readFromUrl() throws IOException {
     Files.write(path, ImmutableList.of("Hello World"), UTF_8);
 
@@ -61,7 +62,7 @@ public class UrlTest {
         .isEqualTo("Hello World" + LINE_SEPARATOR.value());
   }
 
-  @Test
+  @Test @Ignore
   public void readDirectoryContents() throws IOException {
     Files.createDirectory(path);
     Files.createFile(path.resolve("a.txt"));
@@ -72,7 +73,7 @@ public class UrlTest {
     assertThat(Resources.asCharSource(url, UTF_8).read()).isEqualTo("a.txt\nb.txt\nc\n");
   }
 
-  @Test
+  @Test @Ignore
   public void headers() throws IOException {
     byte[] bytes = {1, 2, 3};
     Files.write(path, bytes);
@@ -108,7 +109,7 @@ public class UrlTest {
     }
   }
 
-  @Test
+  @Test @Ignore
   public void contentType() throws IOException {
     path = fs.getPath("foo.txt");
     Files.write(path, ImmutableList.of("Hello World"), UTF_8);
